reana/reana_dev/git.py

```diff
@@ -724,95 +726,98 @@ def git_push(component):  # noqa: D301
             run_command(cmd, component)
 
 
-@git_commands.command(name="git-shared-modules-upgrade")
+@git_commands.command(name="git-upgrade-shared-modules")
 @click.option(
-    "--commit-and-publish",
+    "--component",
+    "-c",
+    multiple=True,
+    default=["CLUSTER", "CLIENT"],
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@click.option(
+    "--use-latest-known-tag",
+    is_flag=True,
+    default=False,
+    help="Should the upgrade use the latest known tag of the shared modules?"
+    " If so, the latest known tag of the shared modules will be used to upgrade the"
+    " provided components. Else, the upgrade will only occur if the latest commit of the"
+    " shared modules points at a tag, in other case, the command will be aborted.",
+)
+@click.option(
+    "--amend",
     is_flag=True,
     default=False,
-    help="Should the changes be committed and pull requests created?"
-    " If so, a commit and a PR with"
-    ' "installation: shared packages version bump" message will'
-    " be created for all affected CLUSTER components.",
+    help="Should the changes be integrated as part of the latest commit of each component?.",
 )
-def git_shared_modules_upgrade(commit_and_publish):
-    """Upgrade all cluster components to latest REANA-Commons version."""
+@click.option(
+    "--push",
+    is_flag=True,
+    default=False,
+    help="Should the feature branches with the upgrades be pushed?.",
```

Comment:
End phrase with `?` instead of `?.`

Reply:
... also, "be pushed to origin?" because elsewhere in other r-dev commands we are pushing to both origin/upstream... so it may help to be more explicit here.